src/components/PokemonList.jsx

```diff
@@ -1,33 +1,55 @@
 import { useState, useEffect } from 'react';
 import axios from 'axios';
+import PropTypes from 'prop-types';
 import Pokemon from './Pokemon';
 import './css/PokemonList.scss';
 
-const PokemonList = () => {
-  /* Declaration of a variable with useState to store the return value of the
-    call to the API (useSate initialized with an empty array) */
+const PokemonList = ({
+  typesFilters,
+  abilityFilters,
+  heightFilters,
+  weightFilters,
+}) => {
   const [pokemons, setPokemons] = useState([]);
 
-  /* Call the API to get the list of pokémons that we want to display on the
-    Homepage (here the 50 first Pokemon) */
   useEffect(() => {
     axios
-      .get('https://pokeapi.co/api/v2/pokemon?limit=50&offset=0')
+      .get('https://pokeapi.co/api/v2/pokemon?limit=100&offset=650')
```

Comment:
Why do you display these specific 100 Pokemon, then ?

Reply:
It's just a random pick for the test, I have change for the first 100 now 🙂